packages/frontend/src/scripts/transaction/submit.ts

```diff
@@ -29,13 +66,8 @@ export async function submit(state: FormState) {
     headers: {
       'Content-Type': 'application/json',
     },
-    body: JSON.stringify(
-      {
-        offer,
-        signature,
-      },
-      (_, value: unknown) =>
-        typeof value === 'bigint' ? value.toString() : value
+    body: JSON.stringify({ offer, signature }, (_, value: unknown) =>
+      typeof value === 'bigint' ? value.toString() : value
```

Comment:
`stringify`ing with `BigInt`s mapped to `string`s is also done `TransactionForm.tsx` - maybe it's worth it to put it into a separate util?